### ISSUE TITLE
Clarify "NIST 800-63B § 5.1.3.3" within "V2.7 Out of Band Verifier"

### ISSUE THREAD

**Issue by cmlh** [OWASP/ASVS#1359]
Insert "_NIST 800-63B_" between "_see_" and "_§ 5.1.3.3_" to clarify the last paragraph of [V2.7 Out of Band Verifier](https://github.com/OWASP/ASVS/blob/master/5.0/en/0x11-V2-Authentication.md#v27-out-of-band-verifier) as per the bold text below:

"_Unsafe out of band authenticators such as e-mail and VOIP are not permitted. PSTN and SMS authentication are currently "restricted" by NIST and should be deprecated in favor of push notifications or similar. If you need to use telephone or SMS out of band authentication, please see **NIST 800-63B** § 5.1.3.3._"

**Comment by elarlang** [comment]
References to NIST corrected (PR #1393).

Instead of examples like `NIST 800-63b` or `NIST 800-63 B` it's correct to use `NIST SP 800-63B`.

I only corrected syntax, I did not validate the meaning - maybe there are references to abstract `NIST SP 800-63` when those should be more precise to `NIST SP 800-63A`, `NIST SP 800-63B` or `NIST SP 800-63C`.

Also, I did not validate, is there correct to use `NIST SP 800-63` or `NIST SP 800-63-3`.

Source: <https://pages.nist.gov/800-63-3/>
* NIST SP 800-63-3 Digital Identity Guidelines
* NIST SP 800-63A Enrollment and Identity Proofing
* NIST SP 800-63B Authentication and Lifecycle Management
* NIST SP 800-63C Federation and Assertions
